perun-base/src/main/resources/perun-roles.yml

```
@@ -2795,58 +2851,77 @@ perun_policies:
         SELF: User
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA:
 
   setSponsoredMember_Vo_User_String_String_User_LocalDate_policy:
     policy_roles:
       - SPONSOR: Vo
         SELF: User
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA:
 
   createSponsoredMembersFromCSV_Vo_String_List<String>_User_policy:
     policy_roles:
       - SPONSOR: Vo
         SELF: User
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA:
 
   group-createSponsoredMembersFromCSV_Vo_String_List<String>_User_policy:
     policy_roles:
       - GROUPADMIN: Group
       - VOADMIN: Vo
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA:
 
   createSponsoredMembers_Vo_String_List<String>_User_policy:
     policy_roles:
       - SPONSOR: Vo
         SELF: User
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA:
 
   setSponsorshipForMember_Member_User_LocalDate_policy:
     policy_roles:
       - SPONSOR: Vo
         SELF: User
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA: Member
+      - MFA: User
 
   unsetSponsorshipForMember_Member_policy:
     policy_roles: []
     include_policies:
       - default_policy
+    mfa_rules:
+      - MFA:
```

```suggestion
      - MFA: Member
      - MFA: User
      - MFA: Vo
```
would make sense to me if `setSponsorshipForMember_Member_User_LocalDate_policy`, `unsetSponsorshipForMember_Member_policy`, `sponsored-sponsorMember_Member_User_LocalDate_policy` and `sponsor-sponsorMember_Member_User_LocalDate_policy` would have the same rules